### PR TITLE
Disable light locker

### LINKED ISSUE #2101: Workspace lock-screen
## :white_check_mark: Checklist

- [x] I have searched open and closed issues for duplicates.
- [x] This is a problem observed when **using** a Data Safe Haven.
- [x] I can reproduce this with the [latest version](https://github.com/alan-turing-institute/data-safe-haven/releases).
- [x] I have read through the [documentation](https://alan-turing-institute.github.io/data-safe-haven/).
- [x] This isn't an open-ended question (open a [discussion](https://github.com/alan-turing-institute/data-safe-haven/discussions) if it is).

## :computer: System information

- **Operating System:** macOS
- **Data Safe Haven version:** develop @ 7b4e6f4
- **Browser details:** Firefox

## :no_entry_sign: Describe the problem

When logging back into a VM with a previously active session, users have to unlock after the login screen, using the same password.

## :steam_locomotive: Workarounds or solutions

This can be disabled with the following (from v4.X.Y)

```
# Disable light-locker which can cause irritating error messages
echo "Hidden=true" >> /etc/xdg/autostart/light-locker.desktop
```

### PR DESCRIPTION
**Pull request by JimMadge**
## :white_check_mark: Checklist

- [ ] You have given your pull request a meaningful title (_e.g._ `Enable foobar integration` rather than `515 foobar`).
- [ ] You are targeting the appropriate branch. If you're not certain which one this is, it should be **`develop`**.
- [ ] Your branch is up-to-date with the **target branch** (it probably was when you started, but it may have changed since then).

### :closed_umbrella: Related issues

~~Closes #2101~~

### CODE REVIEW

File: data_safe_haven/resources/workspace/ansible/desired_state.yaml [hunk 1]
```diff
@@ -170,6 +170,12 @@
         dest: /etc/xdg/xfce4/terminal/
         mode: '0444'
 
+    - name: Disable light locker (screen lock)
+      ansible.builtin.copy:
+        content: "Hidden=true"
+        dest: /etc/xdg/autostart/light-locker.desktop
+        mode: '0644'
+
```
Comment (jemrobinson) [inline data_safe_haven/resources/workspace/ansible/desired_state.yaml:178]: Is it definitely the case that we want to clobber this file? I'm 90% sure it doesn't exist by default but have you checked?

Comment (JimMadge) [reply data_safe_haven/resources/workspace/ansible/desired_state.yaml:178]: It doesn't exist by default :+1:.

**Comment by JimMadge** [comment]
The file has been written correctly.

```console
jim.madge:~$ cat /etc/xdg/autostart/light-locker.desktop 
Hidden=true 
```

**Comment by JimMadge** [comment]
I tested the lock screen and it seems to be working,

- I closed my Guacamole tab
- Opened a new tab and connected to the workspace
- I was prompted for a username and password by the xrdp login
- Went straight back to my desktop session, no need to unlock

**Review by jemrobinson — APPROVED** [review]
LGTM

**Comment by JimMadge** [comment]
Actually, this might not be right. The screen still locks after idling for some time.

**Comment by JimMadge** [comment]
Possibly because the file is missing the `[Desktop Entry]` [group header](https://specifications.freedesktop.org/desktop-entry-spec/latest/basic-format.html)

**Comment by jemrobinson** [comment]
Or it could be that we're using a different lock screen programme - note that the previous instructions were for XFCE3. I think there's now an `xfce4-screenlocker`.

**Comment by JimMadge** [comment]
Hmm, that might be right, `light-locker` isn't installed 😄.

**Comment by JimMadge** [comment]
Probably `xfce-screensaver`, you can trigger the lockscreen with `xfce-screensave-command --lock`.

**Comment by jemrobinson** [comment]
Happy for you to either uninstall it (although I guess this might break a meta-package) or to disable it similarly to light-locker.

**Comment by JimMadge** [comment]
Trying that package in #2128

**Pull request merged**